servers/src/main/java/tachyon/master/MasterSource.java

```diff
@@ -160,43 +195,100 @@ public MetricRegistry getMetricRegistry() {
     return mMetricRegistry;
   }
 
-  public void incCreateFileOps() {
-    mCreateFileOps.inc();
+  public void incCompleteFileOps() {
+    mCompleteFileOps.inc();
+  }
+
+  public void incFilesCompleted() {
+    mFilesCompleted.inc();
+  }
+
+  public void incReleaseFileOps() {
+    mReleaseFileOps.inc();
+
+  }
+
+  public void incFilesReleased(long freeInodes) {
+    mFilesReleased.inc(freeInodes);
+  }
```

Comment:
it seems a bit confusing that some methods take an argument and others do not. Would it be too verbose to include both the +1 and +n methods?